crawl-ref/source/items.cc

```diff
@@ -2614,7 +2614,7 @@ bool drop_item(int item_dropped, int quant_drop)
 
     // If you drop an item in as a merfolk, it is below the water line and
     // makes no noise falling.
-    if (silenced(you.pos()) || you.swimming())
```

Comment:
I removed this "are we silenced" check because that gets checked in prepare_message anyway.